### PR TITLE
Disabling the learner lens still enables navigating to it.

### PR DESCRIPTION
**Pull request by dsjen**
This enables navigating to the learners lens if the functionality is switched off--useful for our beta release. :)

Please review @thallada @ajpal @mulby

### CODE REVIEW

File: analytics_dashboard/courses/views/__init__.py [hunk 1]
```diff
@@ -368,8 +368,10 @@ def get_context_data(self, **kwargs):
         # Get the active primary item and remove it from the list
         primary_nav_item = None
         if self.active_primary_nav_item:
-            primary_nav_item = [i for i in primary_nav_items if i['name'] == self.active_primary_nav_item][0]
-            primary_nav_items.remove(primary_nav_item)
+            navs = [i for i in primary_nav_items if i['name'] == self.active_primary_nav_item]
+            if navs:
+                primary_nav_item = [i for i in primary_nav_items if i['name'] == self.active_primary_nav_item][0]
+                primary_nav_items.remove(primary_nav_item)
```
Comment (mulby) [inline analytics_dashboard/courses/views/__init__.py:374]: a couple of non-blocking nits:

``` python
if len(navs) > 0:  # I think this is more clear?
    primary_nav_items = navs[0]
    primary_nav_items.remove(primary_nav_item)
```

**Comment by mulby** [comment]
:+1:

**Comment by ajpal** [comment]
👍

File: analytics_dashboard/courses/views/__init__.py (hunk 1 above)
Comment (dsjen) [inline analytics_dashboard/courses/views/__init__.py:374]: I totally agree, but will hold off on this for now b/c I'm eager for the build to complete. :)

**Pull request merged**